lisa_commands/request_helper.py

```diff
@@ -203,6 +208,75 @@ def gify_search(media_type, query):
 
     return None
 
+def spotify_search(query):
+    """
+    submit a search to spotify
+    :query what type of content we are querying for and the search terms themselves
+    """
+
+    TOKEN_ENDPOINT = 'https://accounts.spotify.com/api/token'
+    SEARCH_ENDPOINT = 'https://api.spotify.com/v1/search'
+    QUERY_TYPES = ['track', 'album', 'artist', 'playlist']
+
+    query_parts = query.split()
+
+    # Pop first part of query off list, this will be the query type
+    query_type = query_parts.pop(0)
```

Comment:
Does the query not contain the string "spotify me"? If so then you'll need to pop off those words as well

Reply:
It shouldn't because of how this function works: https://github.com/Splyth/slack_bot/blob/master/lisa_commands/commands.py#L115